doc/sphinx/source/notes_5_1.rst

```diff
@@ -1,12 +1,33 @@
-
 **************************
 Release Notes for PoCL 5.1
 **************************
 
+Support for LLVM versions 10 to 13 inclusive has been removed.
+LLVM 14 to 17 are supported.
+
+Support for  `cl_khr_spir` (SPIR 1.x/2.0) has been removed.
+SPIR-V remains supported.
+
 ========================
-Driver-specific features
+New device driver: TBB
 ========================
 
+The TBB device driver uses the Intel oneAPI Threading Building Blocks (oneTBB)
+library for work-group and kernel-level task scheduling. Except for the
+scheduling, the driver is identical to the original CPU driver (pthread).
+
+===========================
+Driver-specific features
+===========================
+
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+CPU driver
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The CPU driver gained support for OpenMP scheduling. Support is
+disabled by default, but can be enabled with CMake option. The
+cpu-minimal driver does not support OpenMP and remains single-threaded.
```

Comment:
```suggestion
The cpu-minimal driver does not support OpenMP and remains single-threaded, serving its purpose as a as simple as possible PoCL driver implemetnation.
```

Reply:
IMO i feel this does not belong here - the release notes are for describing differences between releases, not explaining  existing concepts in detail. I've instead renamed `tbb.rst` to `cpu.rst`, and added a section into `cpu.rst` to describe also 'cpu' and 'cpu-minimal'.